circle.yml

```diff
@@ -6,13 +6,7 @@ test:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
   override:
-    - bin/run-lint :
-        parallel: true
-        files:
-          - client/**/*.js
-          - client/**/*.jsx
-          - server/**/*.js
-          - server/**/*.jsx
+    - bin/run-lint $(git diff --name-only $(git merge-base $(git rev-parse --abbrev-ref HEAD) origin/master)..HEAD *js *jsx)
```

Comment:
This way wouldn’t we run the checks *only* on the changed files? We still want to show problems in other files/lines, but just relegate errors to warnings.